agent/api/task.go

```diff
@@ -248,7 +248,7 @@ func (task *Task) UpdateMountPoints(cont *Container, vols map[string]string) {
 // there was no change
 // Invariant: task known status is the minimum of container known status
 func (task *Task) updateTaskKnownStatus() (newStatus TaskStatus) {
-	seelog.Debug("Updating task: %s", task.String())
+	seelog.Debugf("Updating task: %s", task.String())
```

Comment:
I'd prefer not.  There are differences between the task known status and the container known status and the repetition in the method name helps readability elsewhere when you're dealing with both.